NEWS.rst

```diff
@@ -2,6 +2,35 @@
 NEWS
 ******
 
+Version 0.13
+============
+
+:date: 2017-10-26
```

Comment:
We can keep the date unset or set it to 1970-01-01

Reply:
OK, will do! Thanks.

Reply:
Done.